app/assets/stylesheets/errors.scss

```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the errors controller here.
```

Comment:
Don't need this file too if we don't have any error specific CSS? Even if we do need it, it should be SASS file.